config/hubs/2i2c.cluster.yaml

```diff
@@ -48,10 +48,10 @@ hubs:
           config:
             Authenticator:
               allowed_users: &staging_users
-                - yuvipanda@gmail.com
+                # this will be replaced with the actual list of
+                # staff google ids by the deployer script
+                - staff_google_ids
```

Comment:
This seems reasonable to me - I like the idea that it's explicit. Could we use some special character that is still valid YAML that would make it clear that this will be replaced? Like could we do `{staff_google_ids}` or something? (or would that break YAML somehow?)